apps/taxes/src/modules/avatax/order-created/avatax-order-created-adapter.test.ts

```diff
@@ -1,3 +1,6 @@
-import { describe } from "vitest";
+import { describe, it } from "vitest";
 
-describe.todo("AvataxOrderCreatedAdapter", () => {});
+describe("AvataxOrderCreatedAdapter", () => {
+  // ? how to mock internal call to avatax?
```

Comment:
If you use avatax SDK, you dont, we must assume (on unit-test level) that Avatax is tested. 

So, you can `spyOn(avatax, 'method')` to check if SDK was called with proper payload

On integration/e2e test layer, test scenario can cover if flow `Saleor => App => Avatax => App => Saleor` works, there we need avatax test env